src/code/code_80183070.c

```diff
@@ -1,6 +1,6 @@
 #include "libc64/sleep.h"
 
-void func_80183070(void) {
```

Comment:
With the function named I think we can probably name the file now as well. I would say just `sleep_forever.c` but I think it would be a good idea to let others give opinions before making the change.

Reply:
Yes I hesitated

Reply:
I think renaming the file should be fine